aperturedb/BBoxDataCSV.py

```diff
@@ -60,14 +60,21 @@ class BBoxDataCSV(CSVParser.CSVParser):
     def __init__(self, filename, df=None, use_dask=False):
 
         super().__init__(filename, df=df, use_dask=use_dask)
-        if not use_dask:
```

Comment:
This check ensures that a CSV file is not read at the first pass when dask mode is used. It's a performance optimization, but it's impact can be observed when using a file with millions of rows. I can explain more offline.

Reply:
Thanks, sorry, for got to hit send 🤦 

My interpretation was that this was only making use of the header, which should already be populated by `super().__init__()` and `validation()`. 

In any event, for this PT, I only need the header row.  Is there a way to get that without having to read the whole file?

Reply:
I see. I think we can merge it and worry about the optimization part later.